server/game/player.js

```diff
@@ -670,6 +670,7 @@ class Player extends GameObject {
 
     finalizeForge(key, modifiedCost, cost) {
         this.modifyAmber(-modifiedCost);
+        this.game.raiseEvent('onAmberSpentOnForge', { player: this, amberSpent: cost });
 
         if (this.anyEffect('forgeAmberGainedByOpponent')) {
```

Comment:
Can you remove this block, since "forgeAmberGainedByOpponent" is now deprecated

Reply:
Done. Tests passed as well.